backend/spec/system/question_crud_spec.rb

```diff
@@ -118,6 +125,11 @@
           expect(page).not_to have_selector('.option', text: survey_admin.name)
         end
 
+        it 'shows only teacher options' do
+          find('label', text: 'Option').click
+          expect(page).not_to have_selector('.option', text: option_admin.name)
```

Comment:
add one spec to check that the option_teacher is visible
you only tested thee negative here, positive is missing

Reply:
Done!